frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -56,9 +62,21 @@ const TranslationPage = () => {
   >(new Map());
   const [numTranslatedLines, setNumTranslatedLines] = useState(0);
 
+  const [
+    versionHistoryStack,
+    updateVersionHistoryStack,
+  ] = useState<HistoryStack>({ Undo: [], Redo: [] });
+
   const arrayIndex = (lineIndex: number): number =>
```

Comment:
Not on you, I initially introduced this because I thought pagination would eventually be implemented. It sounds like we're going for infinite scroll, so `lineIndex === translatedStoryLines[0].lineIndex` always. (TL;DR, I abstracted too early, and I think it's worth removing this ASAP bc I'd rather not leave this stale piece to be used and context to be lost)

double-checking @gaoxk, pagination is a no?

Reply:
Woops thought I already replied, pagination is a no